internal/libtor/enabled_test.go

```diff
@@ -253,3 +253,84 @@ func TestControlConnectionExplicitlyClosed(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// This test ensures that we cannot make concurrent calls to the library.
+func TestConcurrentCalls(t *testing.T) {
+	// we need to simulate non zero exit code here such that we're not
+	// actually hitting into the real tor library; by doing this we would
```

Comment:
```suggestion
	// actually hitting into the real tor library; by doing this we
```

*would* is confusing, makes one think this is not actually implemented